addon/utils/plugins/table/tab-input-plugin.ts

```diff
@@ -0,0 +1,105 @@
+import { TabInputPlugin } from '@lblod/ember-rdfa-editor/editor/input-handlers/tab-handler';
+import ModelElement from "@lblod/ember-rdfa-editor/model/model-element";
+import {Manipulation, ManipulationGuidance } from '@lblod/ember-rdfa-editor/editor/input-handlers/manipulation';
+import RawEditor from 'dummy/utils/ce/raw-editor';
+import ModelTable from '@lblod/ember-rdfa-editor/model/model-table';
+
+/**
+ *
+ * @class TableBackspacePlugin
+ * @module plugins/table
+ */
+export default class TableTabInputPlugin implements TabInputPlugin {
+  label = 'backspace plugin for handling table nodes'
+
+  guidanceForManipulation(manipulation: Manipulation, editor: RawEditor) : ManipulationGuidance | null {
+    const selection = editor.selection;
+    if(selection.isInTable) {
```

Comment:
this is always truthful as it has the string "disabled" as value when disabled. As a result the tab handler executor will throw an error every time the plugin is called outside of a table